cmd/podman/network_create.go

```diff
@@ -41,6 +41,7 @@ func init() {
 	flags.IPVar(&networkCreateCommand.Gateway, "gateway", nil, "IPv4 or IPv6 gateway for the subnet")
 	flags.BoolVar(&networkCreateCommand.Internal, "internal", false, "restrict external access from this network")
 	flags.IPNetVar(&networkCreateCommand.IPRange, "ip-range", net.IPNet{}, "allocate container IP from range")
+	flags.StringVar(&networkCreateCommand.MacVLAN, "macvlan", "", "create a macvlan connection based on this device")
```

Comment:
The majority of references, including Docker refs,  that I see for macvlan have an upper case 'M", ala "Macvlan.  We might want to consider that usage.

Reply:
options for us begin with lower case

Reply:
He's talking about the description

Reply:
As @mheon pointed out, the option is fine, it's the description.